docs/use-mainnet/gas-on-linea.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Gas on Linea
+description: Understanding gas on Linea
+---
+
+## What does gas pay for?
+
+Gas must be paid to cover any kind of on-chain computation. For a zk rollup like Linea, gas mainly goes toward the cost of generating proofs for batches of transactions and submitting this data to L1.
```

Comment:
Actually we don't price the proof generation costs right now. We only have two-dimensional pricing: execution/storage on L2 and calldata on L1. In the future, we will add the prover dimension costs to the charged price